index.bs

```diff
@@ -2221,6 +2316,16 @@ spec:css-syntax-3;
   1.  If the new credential type supports {{CredentialMediationRequirement/conditional}} [=user
       mediation=], define `ExampleCredential/isConditionalMediationAvailable()` to return `true`.
 
+  1.  Following the procedure in [[#sctn-registry-requirements]], add an entry for the new
+      "example" [=credential type registry/credential type=] and its corresponding {{CredentialCreationOptions}}' and
+      {{CredentialRequestOptions}}' [=dictionary member=] [=identifier=]&mdash;in this case it
+      is "example"&mdash;to the
+      [[#sctn-cred-type-registry|Credential Type Registry]]:
```

Comment:
This is now missing a mention of the interface object type.

Reply:
Ah, good catch, thanks.  While reviewing the fix for that, I noticed a few other things I/we overlooked and fixed them in 6c82ab2, so re-reviewing might be in order.